### PR TITLE
✨ Added Home functionality to vue

### PR DESCRIPTION
**Pull request by HerrLevin**

### CODE REVIEW

File: resources/lang/de.json [hunk 1]
```diff
@@ -77,27 +77,28 @@
...
+  "user.always-dbl": "Mit #dbl twittern?",
+  "user.private-profile": "Privates Profil",
+  "user.liked-status": "gefällt dieser Status.",
+  "user.liked-own-status": "gefällt der eigene Status.",
+  "user.invalid-mastodon": ":domain scheint keine Mastodon-Instanz zu sein.",
+  "user.no-user": "Keinen Nutzer gefunden.",
+  "dates.-on-": " um ",
+  "dates.decimal_point": ",",
+  "dates.thousands_sep": ".",
+  "dates.April": "April",
```
Comment (MrKrisKrisu) [inline resources/lang/de.json]: What happened here? This would cause conflicts with weblate.

File: resources/lang/en.json [hunk 2]
```diff
@@ -40,27 +40,28 @@
...
+  "user.always-dbl": "Tweet with #dbl?",
+  "user.private-profile": "Private profile",
+  "user.liked-status": "likes this status.",
+  "user.liked-own-status": "likes their own status.",
+  "user.invalid-mastodon": ":domain doesn't seem to be a Mastodon instance.",
+  "user.no-user": "No user found.",
+  "controller.transport.checkin-heading": "Checked in",
+  "controller.transport.checkin-ok": "You've successfully checked into :lineName!|You've successfully checked into line :lineName!",
+  "controller.transport.no-name-given": "You need to provide a station name!",
+  "controller.transport.not-in-stopovers": "Start-ID is not in stopovers.",
```
Comment (MrKrisKrisu) [inline resources/lang/en.json]: What happened here? This would cause conflicts with weblate.

**Review by MrKrisKrisu — CHANGES_REQUESTED** [review]

File: resources/lang/de.json (hunk 1 above)
Comment (HerrLevin) [reply resources/lang/de.json]: Maybe phpstorm reformat? does this really cause conflicts and what should i do to fix this? Just indent it again?

Comment (MrKrisKrisu) [reply resources/lang/de.json]: Yes, just intend. You can enable the .editorconfig to prevent phpstorm to format it like these. It's disabled by default.

**Review by MrKrisKrisu — APPROVED** [review]

**Pull request merged**